gravitee-apim-portal-webui-next/src/components/api-card/api-card.component.spec.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ApiCardComponent } from './api-card.component';
+
+describe('CardComponent', () => {
```

Comment:
Would be good to add a test for the text overflow.

Reply:
I added test for that, but I googled a topic and text overflow can't be tested, it return full content always